karenina/visualization.py

```diff
@@ -331,13 +372,18 @@ def main():
         params['color'] = color
         curr_subject = Individual(subject_id=curr_subject_id,
                                   params=params, \
-                                  metadata={opts.treatment: df[opts.treatment]}, \
+                                  metadata={opts.treatment: row[1][3]}, \
                                   interindividual_variation=.01, verbose=verbose)
         ind.append(curr_subject)
         i+=1
```

Comment:
Minor point on style: 

In the for loop, could we call:

for i,row in enumerate(df.iterrows()):
       #Do stuff

enumerate() helps avoid manual incrementation of i,j, etc.

Reply:
fixed in fit_timeseries and visualization.